lib/test/unit/ui/tap/ext/base_testrunner.rb

```diff
@@ -0,0 +1,62 @@
+require 'test/unit/ui/tap/base_testrunner'
+
+module Test
+  module Unit
+    module UI
+      module Tap
+        class Tap::BaseTestRunner < Test::Unit::UI::TestRunner
```

Comment:
The additional `Tap::` in the class name is confusing me a little. The class is already in the `Tap` module :)

Is it necessary?